### PR TITLE
Dkh group task event update deps

### PR DESCRIPTION
**Pull request by dkhuntrods**
Updating task event dependencies to use the new groupings.
When reviewing, please don't merge as this update will be a breaking change so requires back-end updates in conjunction.

### CODE REVIEW

File: blocks/core/ff_module/ff_module-dropdown-button/ff_module-dropdown-button.js [hunk 1]
```diff
@@ -138,10 +138,10 @@ module.exports = function(options) {
     });
 
     if(!isActivatedForTheseOptions) {
-        if (isBrowser()) console.log('ff_module-dropdown-button is being activated with options: \n',options);
+        // if (isBrowser()) console.log('ff_module-dropdown-button is being activated with options: \n',options);
         activateDropdowns(options);
         activated.push(options);
     } else {
-        if (isBrowser()) console.log('ff_module-dropdown-button has already been activated with options: \n',options);
+        // if (isBrowser()) console.log('ff_module-dropdown-button has already been activated with options: \n',options);
```
Comment (bwfox) [inline blocks/core/ff_module/ff_module-dropdown-button/ff_module-dropdown-button.js]: Any reason not to delete this and the above commented out line?

File: blocks/core/ff_module/ff_module-task-event-repeater/ff_module-task-event-repeater.js [hunk 2]
```diff
@@ -14,26 +15,29 @@ module.exports = React.createClass({
...
+        return events.reduce((memo, event)=> '' + memo + event.localEventId, 'group-');
+    },
+    getGroups: function() {
+        var eventGroups = this.props.eventGroups || [];
+        // if (isArray(events)) {
+        //     events = events.sort(function(a, b) {
+        //         // Reverse chronological order
+        //         return ensureIsDate(b.description.sent) - ensureIsDate(a.description.sent);
+        //     });
+        // }
```
Comment (bwfox) [inline blocks/core/ff_module/ff_module-task-event-repeater/ff_module-task-event-repeater.js]: Don't we still need to apply some sorting? Both on the events in a group and the groups themselves?

**Comment by bwfox** [comment]
Left a couple of comments.

File: blocks/core/ff_module/ff_module-task-event-repeater/ff_module-task-event-repeater.js (hunk 2 above)
Comment (dkhuntrods) [inline blocks/core/ff_module/ff_module-task-event-repeater/ff_module-task-event-repeater.js]: Ah, I was hoping with the grouping that the sorting would be taken care of!

File: blocks/core/ff_module/ff_module-dropdown-button/ff_module-dropdown-button.js (hunk 1 above)
Comment (dkhuntrods) [inline blocks/core/ff_module/ff_module-dropdown-button/ff_module-dropdown-button.js]: I wasn't sure if I wanted it completely gone yet, but yeah I'll just delete it, not like I can't find it again...:)

**Comment by dkhuntrods** [comment]
@bwfox Have updated based on feedback, be much obliged if you could review.

**Comment by bwfox** [comment]
LGTM

**Pull request merged**